Dockerfile

```diff
@@ -45,26 +45,28 @@ COPY dist/eventsource /bin/eventsource
 ENTRYPOINT [ "/bin/eventsource" ]
 
 ####################################################################################################
-# sensor
+# sensor-base
 ####################################################################################################
-FROM alpine:3.12.3 as sensor
+FROM alpine:3.12.3 as sensor-base
 RUN apk update && apk upgrade && \
     apk add --no-cache git
 
-ENV ARGO_VERSION=v2.12.7
+ENV ARGO_VERSION=v3.0.2
 
-COPY --from=base /usr/share/zoneinfo /usr/share/zoneinfo
-COPY --from=base /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
-
-# Argo Workflow CLI
 RUN wget -q https://github.com/argoproj/argo/releases/download/${ARGO_VERSION}/argo-linux-amd64.gz
 RUN gunzip argo-linux-amd64.gz
 RUN chmod +x argo-linux-amd64
 RUN mv ./argo-linux-amd64 /usr/local/bin/argo
 RUN argo version
 
+####################################################################################################
+# sensor
+####################################################################################################
+FROM scratch as sensor
```

Comment:
Good for security.

Suggestion - if you're not confident about what the base image might need - try the distroless images for your bass. They are nearly as secure as scratch - but also have `/tmp` directory and other stuff. I've found that sometime you need this stuff, but it is hard to know if you'lle need it - just some days later you find a weird bug..  I find them more reliable and probably would choose them instead of scratch in the future:

https://github.com/GoogleContainerTools/distroless/blob/main/examples/go/Dockerfile